go/lesson03/README.md

```diff
@@ -112,6 +112,24 @@ tags to the span with some metadata about the HTTP request, and we mark the span
 
 We need to add similar code to the `printHello` function.
 
+#### Handling Errors
+
+Since we turned our single-binary program into a distributed application that makes remote calls, we need to handle errors that may occur during communications. It is a good practice to tag the span with the tag `error=true` if the operation represented by the span failed. So, let's go ahead and update the `format_string` and `print_hello` function with below code snippet:
+
+```go
+resp, err := xhttp.Do(req)
+if err != nil {
+  ext.Error.Set(span, true)
+  span.LogFields(
```

Comment:
I think these can be replaced with just `ext.LogError(span, err)`

https://github.com/opentracing/opentracing-go/blob/17f6344bc92648eeeb74c0555e6a3275aee9e1df/ext/field.go#L9

Reply:
Done. Thank you for pointing on to that.